docs/content/preview/develop/learn/data-types-ycql.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Data types in YCQL
```

Comment:
isn't the problem the title tag is duplicated

Reply:
Right, sorry i got confused.